packages/teams-js/src/private/chat.ts

```diff
@@ -21,61 +21,53 @@ export namespace chat {
    * @hidden
    * Hide from docs
    * --------------
-   * Allows the user to start or continue a conversation with each subentity inside the tab
+   * Allows the user to open a chat with a single user and allows
+   * for the user to specify the message they wish to send.
    *
```

Comment:
We should add openChatRequest to the comments using @param

Reply:
Sorry I promise this is the last one. If we make this:

@param openChatRequest {@link OpenSingleChatRequest} - a request object that contains a user's email as well as an optional message parameter.

Then i believe the documentation will include a link to the definition of OpenSingleChatRequest